src/rana_process_sdk/application/local_test_rana_context.py

```diff
@@ -104,8 +110,13 @@ def lizard_raster_layer_gateway(self) -> LizardRasterLayerGateway:
         provider = LizardApiProvider(lizard_settings=self._settings().lizard)
         return LizardRasterLayerGateway(provider)
 
+    def get_dataset(self, id: str) -> RanaDataset:
+        return self._settings().datasets[id]
+
     def get_lizard_raster_dataset(self, id: str) -> RanaDatasetLizardRaster:
         dataset = self._settings().datasets[id]
+        if not dataset.lizard_raster_id:
+            raise ValueError("Given dataset is not recognized as a Lizard raster")
```

Comment:
Zou hier een ProcessUserError geven. Zou de lizard raster ID niet in de link moeten staan  nu we het nieuwe model gebruiken?